.github/workflows/create-tag-release.yml

```diff
@@ -57,4 +77,65 @@ jobs:
       - name: Push the latest release tag to Quay
         shell: bash
         run: |
-          podman push $QUAY_ODH_DASHBOARD_IMAGE_REPO:${{ github.event.inputs.releaseTag }}
+          podman push $QUAY_ODH_DASHBOARD_IMAGE_REPO:${{ github.event.inputs.RELEASE_VERSION }}-odh
+
+  Create-PR:
+    needs: [Setup, Create-Branch, Tag-Image]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: '0'
+      - uses: fregante/setup-git-user@v2
+      - name: Pull latest release tag ${{ github.event.inputs.RELEASE_VERSION }}-odh from Quay
+        shell: bash
+        run: |
+          podman pull $QUAY_ODH_DASHBOARD_IMAGE_REPO:${{ github.event.inputs.RELEASE_VERSION }}-odh
+      - name: Get full SHA digest for Tag-Image
+        shell: bash
+        run: |
+          echo "DIGEST=$(podman images --format "{{.Digest}}" --filter reference=$QUAY_ODH_DASHBOARD_IMAGE_REPO:${{ github.event.inputs.RELEASE_VERSION }}-odh | sed -n 1p)" >> $GITHUB_ENV
+      - name: Create a branch for the PR
+        shell: bash
+        run: |
+          git checkout -b '${{ github.event.inputs.RELEASE_VERSION }}-PR'
+      - name: Replace odh/params.env version
+        shell: bash
+        run: |
+          sed -i "s/main/${{ env.DIGEST }}/g" manifests/odh/params.env
```

Comment:
Great job here! [PR](https://github.com/manaswinidas/odh-dashboard/pull/2/files) created just fine while testing. We just need to start with `@sha...` instead of `:sha` as per the last [ODH release PR ](https://github.com/opendatahub-io/odh-dashboard/pull/4109)

<img width="933" alt="Screenshot 2025-04-29 at 8 47 28 PM" src="https://github.com/user-attachments/assets/74e3c8db-4fd6-4909-bccc-42f174185e47" />